pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -185,7 +185,7 @@ func (r *ReconcileChaosEngine) Reconcile(request reconcile.Request) (reconcile.R
 		return reconcile.Result{}, err
 	}
 
-	cApp, err := clientset.AppsV1().Deployments(a.namespace).List(metav1.ListOptions{LabelSelector: fmt.Sprintf("%s=%s", labelKey, labelValue), FieldSelector: ""})
+	chaosAppList, err := clientset.AppsV1().Deployments(appInfo.namespace).List(metav1.ListOptions{LabelSelector: fmt.Sprintf("%s=%s", appLabelKey, appLabelValue), FieldSelector: ""})
```

Comment:
Instead of using `fmt.Sprintf("%s=%s", appLabelKey, appLabelValue)` can we use `instance.Spec.Appinfo.Applabel`